docs/wiki/User Guide/Configurations.md

```diff
@@ -245,6 +246,14 @@ We are still trying to improve cruise control. And following are some configurat
 | min.broker.sample.store.topic.retention.time.ms       | Integer | N         | 3600000       | The config for the minimal retention time for Kafka broker sample store topic                                                                                                                           |
 | skip.sample.store.topic.rack.awareness.check          | Boolean | N         | false         | The config to skip rack awareness sanity check for sample store topics                                                                                                                                  |
 
+### MaintenanceEventTopicReader configurations
+| Name                                          | Type    | Required? | Default Value           | Description                                                       |
+|-----------------------------------------------|---------|-----------|-------------------------|-------------------------------------------------------------------|
+| maintenance.event.topic                       | String  | N         | __MaintenanceEvent      | The name of the Kafka topic to consume maintenance events from    |
+| maintenance.event.topic.replication.factor    | Short   | N         | min(2, #alive-brokers)  | The replication factor of the maintenance event topic             |
+| maintenance.event.topic.partition.count       | Integer | N         | 32                      | The partition count of the maintenance event topic                |
```

Comment:
This can probably be a much smaller topic.

Reply:
Changed this to `8`.